test/validation/api/scheduler/scheduler.c

```diff
@@ -141,6 +142,22 @@ static void release_context(odp_schedule_sync_t sync)
 		odp_schedule_release_ordered();
 }
 
+static void scheduler_test_capa(void)
+{
+	odp_schedule_capability_t capa;
+	odp_queue_capability_t queue_capa;
+
+	memset(&capa, 0, sizeof(odp_schedule_capability_t));
+	CU_ASSERT_FATAL(odp_schedule_capability(&capa) == 0);
+	CU_ASSERT_FATAL(odp_queue_capability(&queue_capa) == 0);
+
+	CU_ASSERT(capa.max_ordered_locks != 0);
```

Comment:
I think max ordered locks can be 0 == not supported.

Reply:
Hmm, true.